content/docs/command-reference/exp/run.md

```diff
@@ -94,12 +91,10 @@ the first `dvc exp run`) — useful for re-training ML models, for example.
 
 <details>
 
-### How are checkpoints captured by DVC?
+### How are checkpoints captured?
 
-When DVC runs a checkpoint-enabled experiment, a custom Git branch (in
-`.git/refs/exps`) is started off the repo `HEAD`. A new commit is appended each
-time a checkpoint is registered by the code. These are not pushed to the Git
-remote by default (see `dvc exp push`).
+Instead of a single commit, checkpoint experiments have multiple commits under
+the custom Git reference (in `.git/refs/exps`), forming a branch.
```

Comment:
```suggestion
the custom Git reference (in `.git/refs/exps`), similar to a branch.
```

I think we should be explicit that anything in `refs/exps` is *not* a Git branch. For all intents and purposes, exp refs function the same way, but a formal "Git branch" must be a `refs/heads` reference.

The important distinction being that git CLI commands which expect branch names will not work with (shortened) exp ref names.